addons/sale_mrp/controllers/portal.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+class SaleMrpPortal(CustomerPortal):
+    @http.route(['/my/orders/<int:order_id>'], type='http', auth='public', website=True)
+    def portal_order_page(self, order_id=None, **post):
```

Comment:
I think it's a better idea to add a hook method in portal_order_page in order to add values before the render call

Reply:
Done! I'm not sure if it matters, but I originally copied what `WebsiteSaleDigital.portal_order_page` was doing. It can be updated to use the same hook method, but it's outside of our jurisdiction so I guess we leave it as is?